fulp_modules/container_init/vending_init.dm

```diff
@@ -0,0 +1,7 @@
+/obj/machinery/vending/games/Initialize()
```

Comment:
The file extension is incorrect, meant to be .dm

Reply:
is  not .dm? weird, gonna fix

Reply:
Scratch that, issue on my end